_extras/design.md

```diff
@@ -118,10 +118,84 @@ How do I...
 ### Variables and Assignment (9:15)
 
 *   Teaching: 10 min
+
+Basic assignment of a value.  What kind of thing is it?  Can assign results
+of computation and use a variable on the right to assign to the left.
+
+```
+a = 3
+print(a)
+type(a)
+a = a + 60
+print(a)
+```
+
+Things can change type and operators can change 'meaning'
+
+```
+a = 'gdp.csv'
+print(a)
+type(a)
+a = 'data/' + a
+print(a)
+```
+
+Use meaningful names for variables.  For example, time
+
+```
+minutes = 3
+seconds = 12
+seconds = seconds + minutes*60
+print(seconds)
+```
+
 *   Exercises: 10 min
+
+Qu:  What kind of thing is 3.4?
+Ans:
+
+```
+     min = 3.25
+     type(3.25)
```

Comment:
This `print('type of 3.25 is:', type(3.25))` is certainly better, but how much about the `print()` function do we explain, where, and when?  It would be good to show the learners early, but I think it should get an item in the outline to explain `print()` if it's going to get used, yes?